frodo-runtime/src/main/java/com/fernandocejas/frodo/internal/DebugLog.java

```diff
@@ -18,6 +21,28 @@
    * @param message The message you would like logged.
    */
   public void log(String tag, String message) {
-    Log.d(tag, message);
+    if (!runningJUnit())
```

Comment:
I believe it would be better to just catch all `Exceptions` as it won't slow down execution of unit tests as much as checking the stack trace. Moreover, this implementation is tied to JUnit. What if we are just running our code on desktop?

Reply:
i need  help to resolved bug java.lang.RuntimeException: Method e in android.util.Log not mocked